File: src/cn/vove7/plugin/rest/filetype/RestClientPlugin.java
package cn.vove7.plugin.rest.filetype;

import com.intellij.openapi.components.ApplicationComponent;
import org.jetbrains.annotations.NotNull;

/**
 * @author danblack
 */
public class RestClientPlugin implements ApplicationComponent {

    @NotNull
    @Override
    public String getComponentName() {
        return "Rest client plugin";
    }

    @Override
    public void initComponent() {
    }

    @Override
    public void disposeComponent() {
    }
}

File: src/main/kotlin/cn/vove7/plugin/rest/filetype/RestClientPlugin.kt
package cn.vove7.plugin.rest.filetype

import com.intellij.openapi.components.ApplicationComponent

/**
 * @author danblack
 */
class RestClientPlugin : ApplicationComponent {
    override fun getComponentName(): String {
        return "Rest client plugin"
    }

    override fun initComponent() {}
    override fun disposeComponent() {}
}